tests/std/tests/P0896R4_ranges_subrange/test.compile.pass.cpp

```diff
@@ -83,6 +83,10 @@ namespace test_view_interface {
 
         S end();
         S end() const requires (to_bool(HasConstRange));
+
+        unsigned int size() requires (to_bool(Diff) && !std::derived_from<Cat, forward_iterator_tag>);
+        unsigned int size() const requires (to_bool(HasConstRange) && to_bool(Diff) &&
+            !std::derived_from<Cat, forward_iterator_tag>);

```

Comment:
Style nitpick: We conventionally put binary operators at the beginning of the line, but clang-format is suppressed here.